package/features/environment.py

```diff
@@ -26,6 +26,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+#------------------------------------------------------------------------------
+# Copyright (c) 2013, Nucleic Development Team & H. Rutjes.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#------------------------------------------------------------------------------
+
```

Comment:
It'd make a lot more sense to me for us to only put this legal header on the one JS file that includes kiwi.js, and none of the Python files, as they don't even include kiwi.js.

Reply:
Yeah I thought this seemed a little unusual too, after discussing with @yetudada here are the changes:

- reverted the notice from Python files (see in f92d68f) 
- added the notice as a special `@license` comment where it is imported in the JS source (see [here](https://github.com/quantumblacklabs/kedro-viz/pull/185/commits/f92d68fbf2d07e8d82ac3ce55e12274d05269259#diff-506cc426df42a78b3776a93989f292c6)) so this way it should also be retained in build files.

Does this suit all?